python/surf/axi/_AxiVersion.py

```diff
@@ -258,13 +258,13 @@ def parseBuildStamp(var,read):
        
 
     def hardReset(self):
-        print('AxiVersion hard reset called')
+        print('%s hard reset called' % self)
```

Comment:
Sorry to nitpick, but this is considered improper formatted string style for python3
use `f'{self.path} hard reset called'`

Reply:
Resolved in https://github.com/slaclab/surf/commit/c48495f05177027fc23d6090cd8899a45073e976